### ISSUE TITLE
Some training detail problems

### ISSUE THREAD

**Issue by PengWenChen** [MichiganCOG/TASED-Net#8]
Hello.

First, many thanks for the generous sharing of the code.

I'm trying to reproduce the training on DHF1K (1~600) now. But I didn't find the evaluation code for validation set with evaluation metrics noticed in the paper.
Could you please also share the code of evaluation?

Thank you.

**Comment by kylemin** [comment]
Hi,
Thank you for your interest.

We will update our code in the future, but in the meanwhile,
please kindly refer to the following two codes:
[Python](https://github.com/tarunsharma1/saliency_metrics/blob/master/salience_metrics.py)
[MATLAB](https://github.com/wenguanwang/DHF1K/blob/master/code_for_Metrics.zip)

I hope it helps.
Thank you.

**Comment by PengWenChen** [comment]
Thank you for your kindly reply.
I'll try it first, thanks!
